packages/metal-dom/test/dom.js

```diff
@@ -359,6 +360,73 @@ describe('dom', function() {
 			dom.triggerEvent(element, 'click');
 			assert.strictEqual(0, listener.callCount);
 		});
+
+		it('should listen to keyboard events by keyCode alias', function() {
+			var element = document.createElement('input');
+			element.setAttribute('type', 'text');
+			dom.enterDocument(element);
+
+			var listener = sinon.stub();
+			var handle = dom.on(element, 'keydown:enter', listener);
+			dom.triggerEvent(element, 'keydown', {keyCode: KEYMAP['ENTER']});
```

Comment:
Dot notation is prefered whenever possible when using objects, so this should be `KEYMAP.ENTER`.